app/services/object_file_validator.rb

```diff
@@ -60,6 +61,10 @@ def object_has_hierarchy?
     filepaths.any?(/\/+/)
   end
 
+  def object_equals_druid?
+    filepaths.map { |filepath| filepath.split('/') }.flatten.any?(druid.druid.delete_prefix('druid:'))
```

Comment:
I think this is only supposed to be (1) if the first directory is the bare druid or (2) the file is in the base directory and is named the bare druid.

See https://github.com/sul-dlss/cocina-models/blob/main/lib/cocina/models/validators/reserved_filename_validator.rb#L37

Reply:
@andrewjbtw can you confirm this is only in the base directory? I'll update.

Reply:
Thank you, updated.